features/child_workflow/signal/ChildWorkflow.java

```diff
@@ -0,0 +1,15 @@
+package child_workflow.signal;
+
+import io.temporal.workflow.SignalMethod;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+
+@WorkflowInterface
+public interface ChildWorkflow {
```

Comment:
Let's throw this interface as an inner interface in that `feature` class to if we can

Reply:
Done.